dbt_schema_builder/schema_builder.py

```diff
@@ -85,8 +90,13 @@ def handle(args):
     parsed = parse_args(args)
 
     if parsed.command == "build":
-        task = SchemaBuilderTask(parsed)
-        task.run()
+        if parsed.nopii is True:
```

Comment:
Commit lint is complaining about this, typically in python you won't even test booleans, just use `if parsed.nopii:`. Personally I prefer the explicit check so I agree with you here, but the repo standards are complaining in the automated checks so 🤷 . You could also do `task.run(no_pii=parsed.nopii)` Though I forget if argparse treats `action='store_true'` as a default of `False` or `None`